openbas-framework/src/main/java/io/openbas/execution/ExecutionExecutorService.java

```diff
@@ -115,4 +144,9 @@ private void launchExecutorContextForAsset(Inject inject, Asset asset) {
       }
     }
   }
+
+  @Autowired
+  public void setInjectStatusRepository(InjectStatusRepository injectStatusRepository) {
```

Comment:
Autowired function that does initialization would be better at the beginning of the class. Besides, the class have a RequiredArgsConstructor annotation so all these Autowired methods should go (you just need to set the services and repository to final).